File: cpp/242.cpp
class Solution {
public:
  bool isAnagram(string s, string t) {
    vector<int> vals(26,0);
  for (int i = 0; i < s.length(); i++)
      vals[s[i] - 'a']++;
  for (int k = 0; k < t.length(); k++)
      vals[t[k] - 'a']--;
    for (int x : vals)
      if (x != 0) {return false;}
    return true;
  }
};

File: py/242.py
class Solution:
    def isAnagram(self, s:str, t:str) -> bool:
        vals = [0] * 26
        for x in s:
            vals[ord(x) - 97] += 1
        for y in t:
            vals[ord(y) - 97] -= 1
        return all(x == 0 for x in vals)
